Wei/Model/HTTPRequest.swift

```diff
@@ -0,0 +1,56 @@
+//
+//  HTTPRequest.swift
+//  Wei
+//
+//  Created by Ryo Fukuda on 2018/07/17.
+//  Copyright © 2018 yz. All rights reserved.
+//
+
+import APIKit
+
+struct HTTPRequest<Request: WeiRequest>: APIKit.Request {
```

Comment:
Do you have any plan for this? I mean `HTTPRequest` is as same as `WeiRequest` except adding `Authorization` header. or the name of `HTTPRequest` should be changed to meaningful name like... `AuthorizationRequest` ??